pandas/tests/test_base.py

```diff
@@ -347,27 +347,28 @@ def test_ops(self):
                     expected = expected.astype('M8[ns]').astype('int64')
                     assert result.value == expected
 
-    def test_nanops(self):
+    @pytest.mark.parametrize('klass', [Index, Series])
+    @pytest.mark.parametrize('op', ['max', 'min'])
+    def test_nanops_max_min(self, op, klass):
```

Comment:
Hmm, I thought I did this parametrization in https://github.com/pandas-dev/pandas/pull/18876. I can move it there if you want.

Reply:
Actually, these seem to be a remnant of an earlier version where I was adding `min_count` everywhere. I'll just remove it.